templates/blocks/blok_dossier.twig

```diff
@@ -1,38 +1,45 @@
-<div class="bg-gray-800 text-white">
+<div class="bg-gray-800 text-white my-4">
+    <div class="lg:hidden">
+        <div class="aspect-w-21 aspect-h-9">
+            <div>
+                <img
+                    class="object-cover absolute inset-0 w-full h-full"
+                    src="{{ Image(block.term.dossier_afbeelding_breed).src|thumbor(1024, 439) }}"
+                    srcset="{{ Image(block.term.dossier_afbeelding_breed).src|thumbor(1024*2, 439*2) }} 2x"
```

Comment:
Wordt op 1x devices de juist image getoond? Of moet de srcset ook een 1x entry hebben?

Reply:
Volgens het meest recente voorbeeld van mdn niet. Dit zit door de hele site zo. 

https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#resolution_switching_same_size_different_resolutions

https://mdn.github.io/learning-area/html/multimedia-and-embedding/responsive-images/srcset-resolutions.html